5-app-infra/README.md

```diff
@@ -50,64 +44,594 @@ Hub and Spoke network model. It also sets up the global DNS hub</td>
+   echo "remote_state_bucket = ${remote_state_bucket}"
+   sed -i "s/REMOTE_STATE_BUCKET/${remote_state_bucket}/" ./common.auto.tfvars
+   ```
+
+5. Update `backend.tf` with your bucket from the infra pipeline output.
+
+   ```bash
+   export backend_bucket=$(terraform -chdir="../gcp-projects/business_unit_3/shared/" output -json state_buckets | jq '."bu3-service-catalog"' --raw-output)
+   echo "backend_bucket = ${backend_bucket}"
+
+   for i in `find -name 'backend.tf'`; do sed -i "s/UPDATE_APP_INFRA_BUCKET/${backend_bucket}/" $i; done
+   ```
+
+6. Commit changes.
+
+   ```bash
+   git add .
+   git commit -m 'Initialize repo'
+   ```
+
+7. Push your plan branch to trigger a plan for all environments. Because the
+   _plan_ branch is not a [named environment branch](../docs/FAQ.md#what-is-a-named-branch), pushing your _plan_
+   branch triggers _terraform plan_ but not _terraform apply_. Review the plan output in your Cloud Build project https://console.cloud.google.com/cloud-build/builds;region=DEFAULT_REGION?project=YOUR_INFRA_PIPELINE_PROJECT_ID
+
+   ```bash
+   git push --set-upstream origin plan
+   ```
+
+8. Merge changes to shared. Because this is a [named environment branch](../docs/FAQ.md#what-is-a-named-branch),
+   pushing to this branch triggers both _terraform plan_ and _terraform apply_. Review the apply output in your Cloud Build project https://console.cloud.google.com/cloud-build/builds;region=DEFAULT_REGION?project=YOUR_INFRA_PIPELINE_PROJECT_ID
+
+   ```bash
+   git checkout -b shared
+   git push origin shared
+   ```
+
+9. `cd` out of the `bu3-service-catalog` repository.
+   
+   ```bash
+   cd ..
+   ```
+
+#### Configuring Cloud Source Repository of Service Catalog Solutions Pipeline
+
+1. Grab the Service Catalogs ID
+
+   ```bash
+   export SERVICE_CATALOG_PROJECT_ID=$(terraform -chdir="gcp-projects/business_unit_3/shared" output -raw service_catalog_project_id)
+   echo ${SERVICE_CATALOG_PROJECT_ID}
+   ```
+
+2. Clone the freshly minted Cloud Source Repository that was created for this project.
+   
+   ```bash
+   gcloud source repos clone service-catalog --project=${SERVICE_CATALOG_PROJECT_ID}
+   ```
+
+3. Enter the repo folder and copy over the service catalogs files from `5-app-infra/source_repos/service-catalog` folder.
+   
+   ```bash
+   cd service-catalog/
+   cp -RT ../terraform-google-enterprise-genai/5-app-infra/source_repos/service-catalog/ .
+   ```
+
+4. Commit changes and push main branch to the new repo.
+   
+   ```bash
+   git add .
+   git commit -m 'Initialize Service Catalog Build Repo'
+
+   git push --set-upstream origin main
+   ```
+
+5. `cd` out of the `service_catalog` repository.
+   
+   ```bash
+   cd ..
+   ```
+
+6. Navigate to the project that was output from `${ARTIFACT_PROJECT_ID}` in Google's Cloud Console to view the first run of images being built.
+
+### Run Terraform locally
```

Comment:
Run Terraform locally section is outdated

You have two options:
- Remove it entirely and assume the user will always run cloud build
- Update it with the missing steps